css-paint-api/Overview.bs

```diff
@@ -131,10 +131,16 @@ partial interface CSS {
 
 The {{PaintWorkletGlobalScope}} is the global execution context of the {{paintWorklet}}.
 
+The {{PaintWorkletGlobalScope}} has a {{PaintWorkletGlobalScope/devicePixelRatio}} property,
+which represents the ratio of the resolution in physical pixels to the resolution of CSS
+pixels for the current display device. When measured from the same display device, this
+value must be identical to the Window.devicePixelRatio.
```

Comment:
I was going to ask to reference devicePixelRatio, but it looks to me like the csswg has [resolved not to standardize devicePixelRatio](https://github.com/w3c/csswg-drafts/issues/417#issuecomment-267111094), see also this blog post for [unprefixing -webkit-device-pixel-ratio](https://www.w3.org/blog/CSS/2012/06/14/unprefix-webkit-device-pixel-ratio/). If the working group hasn't changed their stance on this, perhaps it's worth a note that we should aim to track whatever the expected way of querying the display density from javascript is. That thread suggests that the standard way is resolution but as far as I can tell you're unable to query that from javascript.